### ISSUE TITLE
Nix support

### ISSUE THREAD

**Issue by cafetestrest** [Aylur/dotfiles#38]
Will there be any support for nix and nixos?
I'm running nixos and trying to figure out how to get it all working there :)

**Comment by Aylur** [comment]
I did try to make it work with only home-manager but couldn't and eventually gave up. I guess your issue is with the js scripts aswell.

**Comment by cafetestrest** [comment]
Yeah, well mostly with my non-existent knowledge of glib and how to install its dependencies on nixos.
I can make it work without throwing any errors, but then I tend to disable upowerglib, gnomebluetooth and NM in lib.js.
Also have to disable a lot of widgets for it to stop giving me errors, when all the errors disappeared I was left with empty bar.
I did make app launcher to work though.

I really dig the bar and it's set of functionalities, just frustrating to make it work fully on nix.

**Comment by isabelroses** [comment]
To get gjs to not have any kind of type errors you need to alter the gjs package.
This is how i did it https://github.com/isabelroses/nur/blob/master/pkgs/gjs/default.nix.
